.ci/magician/cmd/check_cassettes.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"magician/vcr"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// TODO(trodge): Move this into magician/github along with repo cloning
+const githubUsername = "modular-magician"
+
+var environmentVariables = [...]string{
+	"GITHUB_TOKEN",
+	"GOCACHE",
```

Comment:
GOCACHE shouldn't be needed... Lets try to run the dependent command in bash

Reply:
same with GOPATH

Reply:
I could try running `make testacc` in a bash stub, but I'm not sure how it would be different from the way I'm running it right now. I'm also not sure how the arguments would need to be escaped if they're being passed to bash instead of directly to `make`.